docs/zkapps/tutorials/03-deploying-to-a-network.mdx

```diff
@@ -34,7 +34,7 @@ Mina zkApps are available only on feature-complete Berkeley, Mina's public Testn
 
 ## Prerequisites
 
-This tutorial has been tested with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.10.0` and [SnarkyJS](https://www.npmjs.com/package/snarkyjs) `0.11.3`.
+This tutorial has been tested with [Mina zkApp CLI](https://github.com/o1-labs/zkapp-cli) version `0.13.0` and [o1js](https://www.npmjs.com/package/o1js) `0.12.1`.
```

Comment:
if you have tested with 12.1, great! We have not officially tested with o1js, so the SnarkyJS release 11.0 is accurate